doc/psyclone_script.rst

```diff
@@ -88,67 +87,70 @@ Choosing the API
-running the generator.py script with the -h option.
+running the ``psyclone`` script with the ``-h`` option.
 
 If you use a particular API frequently and it is not the default then
 you can change the default by creating a copy of the default
 ``psyclone.cfg`` file and editing it. See :ref:`configuration` for
 more details.
 
 If your code uses an API that is different to the default then you can
-specify this as an argument to the generator.py script.
+specify this as an argument to the ``psyclone`` script.
 ::
 
-    > python <PSYCLONEHOME>/src/generator.py -api dynamo0.1 alg.f90
+    > psyclone -api dynamo0.1 alg.f90
 
 File output
 -----------
 
 By default the modified algorithm code and the generated PSy code are
-output to the terminal. These can be output to a file by using the
--oalg <file> and -opsy <file> options respectively. For example, the
+output to the terminal. These can instead be output to files by using the
+``-oalg <file>`` and ``-opsy <file>`` options, respectively. For example, the
 following will output the generated psy code to the file 'psy.f90' but
 the algorithm code will be output to the terminal:
 ::
 
-    > python <PSYCLONEHOME>/src/generator.py -opsy psy.f90 alg.f90
+    > psyclone -opsy psy.f90 alg.f90
 
 Algorithm files with no invokes
 -------------------------------
 
-If the generator script is provided with a file that contains no
+If the ``psyclone`` script is provided with a file that contains no
 ``invoke`` calls then the script outputs a warning to ``stdout`` and
 copies the input file to ``stdout``, or to the specified algorithm
-file (if the -oalg <file> option is used). No PSy code will be
-output. If a file is specified using the -opsy <file> option this file
+file (if the ``-oalg <file>`` option is used). No PSy code will be
+output. If a file is specified using the ``-opsy <file>`` option this file
 will not be created.
-::
 
-    > python <PSYCLONEHOME>/src/generator.py -opsy psy.f90 -oalg alg_new.f90 empty_alg.f90
+.. code-block:: bash
+
+    > psyclone -opsy psy.f90 -oalg alg_new.f90 empty_alg.f90
     Warning: 'Algorithm Error: Algorithm file contains no invoke() calls: refusing to
     generate empty PSy code'
 
 Kernel directory
 ----------------
 
 When an algorithm file is parsed, the parser looks for the associated
-kernel files. The way this is done requires that any kernel routine
-specified in an invoke must have an explicit use statement. For
+kernel files. The way in which this is done requires that any kernel routine
+called within an invoke must have an explicit use statement. For
 example, the following code gives an error:
-::
+
+.. code-block:: bash
 
     > cat no_use.f90
     program no_use
       call invoke(testkern_type(a,b,c,d))
     end program no_use
-    > python <PSYCLONEHOME>/src/generator.py no_use.f90
+    > psyclone no_use.f90
     "Parse Error: kernel call 'testkern_type' must be named in a use statement"
```

Comment:
The `Parse Error` message needs to be updated.

When running the `no_use.f90` example, I get the following error for all APIs except `dynamo0.3`
```
"Parse Error: kernel call 'testkern_type' must either be named in a use statement or be a recognised built-in (one of '[]' for this API)"
```
whereas in `dynamo0.3` I get the list of built-ins in place of brackets, i.e. 
```
"Parse Error: kernel call 'testkern_type' must either be named in a use statement or be a recognised built-in (one of '{'inc_x_times_y': <class 'psyclone.dynamo0p3_builtins.DynIncXTimesYKern'>, ...' for this API)"
```

Reply:
Good point. I've updated the example and expanded the text a little to mention that this `use` requirement does not apply to Built-ins.